### PR TITLE
Add a set balance mocking function

### PR DESCRIPTION
**Pull request by samliok**

### CODE REVIEW

File: x/programs/test/programs/unit_test/Cargo.toml
Comment (samliok) [inline x/programs/test/programs/unit_test/Cargo.toml:1]: thinking we could have a unit testing crate that would test our context mocks and other stuff compiled with `#[test]`

File: x/programs/test/programs/test_directive/Cargo.toml
Comment (samliok) [inline x/programs/test/programs/test_directive/Cargo.toml:1]: thinking this crate would be for testing `wasmlanche` when compiled with the `test` feature

Comment (richardpringle) [reply x/programs/test/programs/test_directive/Cargo.toml:1]: Not sure I understand? Usually crates in the /x/programs/test/ directory are for testing the runtime.

Comment (samliok) [reply x/programs/test/programs/test_directive/Cargo.toml:1]: hmm yea, what would be a better place to put this? in the `tests` folder of wasmlanche?

Comment (richardpringle) [reply x/programs/test/programs/test_directive/Cargo.toml:1]: Yeah, there's already a test-crate in there

**Review by richardpringle — APPROVED** [review]

**Pull request merged**